koku/api/iam/test/test_customer_manager.py

```diff
@@ -0,0 +1,141 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Test the Customer Manager."""
+
+from django.contrib.auth.models import Group
+
+from api.iam.customer_manager import (CustomerManager, CustomerManagerError, CustomerManagerPermissionError)
+from api.iam.models import Customer, User
+from api.iam.serializers import (CustomerSerializer, UserSerializer)
+from api.provider.models import Provider
+from .iam_test_case import IamTestCase
+
+
+class CustomerManagerTest(IamTestCase):
+    """Tests for Customer Manager."""
+
+    def setUp(self):
+        """Set up the Customer Manager tests."""
+        super().setUp()
+        self.create_service_admin()
+
+    def tearDown(self):
+        """Tear down the Customer Manager tests."""
+        super().tearDown()
+        User.objects.all().delete()
```

Comment:
I know you are just following suit with other tearDown methods, but it turns out the Django TestCase wraps each test in its own transaction, so deleting objects is superfluous. They just get rolled back after each test anyway.

Reply:
good to know!